### ISSUE TITLE
Deadlock when using UVC camera with Nougat RC

### ISSUE THREAD

**Issue by FROGbots-4634** [FIRST-Tech-Challenge/SkyStone#12]
I just noticed this in the release notes from v4.x:

> There might be a possible deadlock which causes the RC to become unresponsive when using a UVC webcam with a Nougat Android Robot Controller.

I can indeed confirm this is the case, as I did in fact have this happen occasionally during last season.

**Comment by ftctechnh** [comment]
@FROGbots-4634 
Frogbots - are you able to reproduce the problem?  I believe I have seen this but intermittently/randomly and have not been able to reproduce it regularly.

**Comment by FROGbots-4634** [comment]
> @FROGbots-4634
> Frogbots - are you able to reproduce the problem? I believe I have seen this but intermittently/randomly and have not been able to reproduce it regularly.

Yeah I can't reliably reproduce it either. It's very random, but I can definitely say it _does_ happen.

**Comment by FROGbots-4634** [comment]
@ftctechnh  FWIW, I've run into this a few times again (and on another device running Nougat as well, so I know it's not just the device) and while I still can't reliably reproduce, it seems to happen most often when deploying the RC app from AS, and then right after it launches plugging in the webcam. The RC app doesn't pick up the USB device and when I hit restart robot it hangs.
